### ISSUE TITLE
Minimum requirement of iOS for MTLManagedObjectAdapter

### ISSUE THREAD

**Issue by kevinnguy** [CocoaPods/Specs#13202]
Hi @sodastsai , I was wondering why the minimum requirement of iOS version for MTLManagedObjectAdapter is iOS 8.0 and Mantle's minimum requirement is iOS 5.0. Is it possible to lower MTLManagedObjectAdapter's minimum requirement? Thanks!

I also presented this question in MTLManagedObjectAdapter repo
https://github.com/Mantle/MTLManagedObjectAdapter/issues/12

**Comment by sodastsai** [comment]
Cuz this adapter is for Mantle 2.x., and the Mantle team said the 2.x is [for OS X 10.9 and iOS 8](https://github.com/Mantle/Mantle/blob/master/CHANGELOG.md#os-x-109-and-ios-8).  So I set its minimum deploy target to the same as Mantle 2.x. 

I'm not sure whether old iOS works or not. If it does works, I could lower it. :)

**Comment by kevinnguy** [comment]
I'm currently working on a project that supports iOS 7 and haven't gotten any warnings. I'd assume since Mantle supports iOS 5.0 in its podspec that MTLManagedObjectAdapter would be the same as well. Let me see if Mantle kept the same minimum when it still had MTLManagedObjectAdapter in its class.

**Comment by kevinnguy** [comment]
https://github.com/CocoaPods/Specs/blob/master/Specs/Mantle/1.5.4/Mantle.podspec.json

Mantle had its minimum requirement at iOS 5 so I would think we can lower MTLManagedObjectAdapter's requirement. But since the Mantle team said that 2.x is for OS X 10.9 and iOS 8, I'm confused with Mantle's podspec.

**Comment by sodastsai** [comment]
Well, by the policy listed in #12199, to change the content of a podspec, you should push a new version of the package. But pushing a new version of a package requires add a new tag related to the version in the code repository. (a CocoaPods warning: `The version should be included in the Git tag.`) But the git tag is not something I could change.

tl;dr, if you need use it directly via `pod 'MTLManagedObjectAdapter'`, you have to ask the author of its git repository to add a new tag for this modification.

Or , you could use following podspec content by
1. save it as a file in your code folder
2. use it by `pod 'MTLManagedObjectAdapter', :podspec => '/path/to/the/file.podsec'`


```ruby
Pod::Spec.new do |s|
  s.name     = 'MTLManagedObjectAdapter'
  s.version  = '1.0-1'
  s.license  = 'MIT'
  s.summary  = 'Model framework for Cocoa and Cocoa Touch.'
  s.homepage = 'https://github.com/Mantle/Mantle'
  s.authors  = { 'GitHub' => 'support@github.com' }
  s.source   = { :git => 'https://github.com/Mantle/MTLManagedObjectAdapter.git', :tag => "1.0" }
  s.requires_arc = true

  s.ios.deployment_target = '5.0'
  s.osx.deployment_target = '10.7'

  s.source_files = 'MTLManagedObjectAdapter/MTLManagedObjectAdapter.{h,m}'
  s.public_header_files = 'MTLManagedObjectAdapter/MTLManagedObjectAdapter.h'

  s.subspec "extobjc" do |ss|
    ss.source_files = 'MTLManagedObjectAdapter/extobjc/*.{h,m}'
    ss.private_header_files = 'MTLManagedObjectAdapter/extobjc/*.h'
  end

  s.dependency 'Mantle', '~> 2.0'
  s.frameworks = 'Foundation', 'CoreData'

  # Patches for header map and header search order (clashes with `Mantle`)
  s.prepare_command = <<-CMD
  PREFIX="mtl_moa_"
  # Add prefix to header imports
  ext_header_prefix_src() {
    SOURCE_FILE=$1
    EXT_HEADER_NAME=$2
    sed -i.bak "s/\"${EXT_HEADER_NAME}\"/\"${PREFIX}${EXT_HEADER_NAME}\"/g" ${SOURCE_FILE} && rm ${SOURCE_FILE}.bak
  }
  ext_header_prefix_src MTLManagedObjectAdapter/MTLManagedObjectAdapter.m EXTRuntimeExtensions.h
  ext_header_prefix_src MTLManagedObjectAdapter/MTLManagedObjectAdapter.m EXTScope.h
  ext_header_prefix_src MTLManagedObjectAdapter/extobjc/EXTRuntimeExtensions.m EXTRuntimeExtensions.h
  ext_header_prefix_src MTLManagedObjectAdapter/extobjc/EXTScope.m EXTScope.h
  # Change header name
  ext_header_prefix_mv() {
    SOURCE_FILE=$1
    FILE_NAME=`basename ${SOURCE_FILE}`
    DIR_NAME=`dirname ${SOURCE_FILE}`
    mv ${SOURCE_FILE} `dirname ${SOURCE_FILE}`/${PREFIX}`basename ${SOURCE_FILE}`
  }
  export -f ext_header_prefix_mv
  export PREFIX=${PREFIX}
  find MTLManagedObjectAdapter/extobjc -name "*.h" -exec bash -c 'ext_header_prefix_mv "$0"' {} \\;
  unset ext_header_prefix_mv
  unset PREFIX
  CMD
end
```

**Comment by segiddins** [comment]
@sodastsai you totally can push a new spec version that references the same tag

**Comment by sodastsai** [comment]
Well well well ... I tried and said at last comment. 

To update a new spec version, maybe "1.0-1" or "1.1", the name of the git tag must contain also "1.0-1" or "1.1". The `trunk` repo of Cocoapods applies more restrict rules.

**Comment by segiddins** [comment]
Closing as this is not an issue for the specs repo.
